### DISCUSSION TITLE
X forwarding from linux cluster to Windows

### DISCUSSION THREAD

**Discussion by andrembcosta** [pyvista/pyvista#3896] [Q&A]
Hi all, 

I have just started using pyvista to post-process the results of my finite element calculations and yesterday, for the first time, I was trying to use it in a cluster and forward the output to my local machine display. 

I saw there are some instructions about installing libgl1-mesa-dev and xvfb for remote usage, but I was not able to install those without sudo rights (which I don't have in this cluster). However, they have some version of mesa in this cluster and I was able to get the forward the graphics to my personal linux machine. 

However, I also wanted to do the same in my windows laptop, and while I was also able to set up x forwarding using vcXsrv and run some simple graphics programs from that cluster (like xeyes and xclock), I was not able to get output from pyvista. Instead, I get an error saying 

Xlib: extension "MIT-SHM" missing on display "127.0.0.1:10.0". 

I am not sure if this is actually a pyvista issue, or a matplotlib problem, or even some setup problem in my windows machine, but since I am able to run xclock just fine, I think this would be the best place to begin searching for some help.

Has anyone used pyvista in this same context? Maybe some idea of what this problem could be?

**Comment by banesullivan** [comment]
Hi @andrembcosta! So cool to hear you got that set up for your linux-linux set up. I have no idea how to go about this for a windows machine and would recommend you post on the VTK discourse as there may be more people there who have experience with this: https://discourse.vtk.org/

Definitely follow up if you get it working!
